strongbox-security/strongbox-user-management/src/main/resources/etc/conf/strongbox-authorization.xml

```diff
@@ -1,49 +1,28 @@
 <?xml version="1.0" standalone="yes"?>
 <authorization-configuration>
-    <roles>
-        <!-- special treatment role -->
-        <role>
-            <name>ANONYMOUS_ROLE</name>
-            <description>Common anonymous user role</description>
-            <privileges>
-                <privilege>ARTIFACTS_RESOLVE</privilege>
-                <privilege>SEARCH_ARTIFACTS</privilege>
-            </privileges>
-        </role>
-        <role>
-            <name>USER_ROLE</name>
-            <description>Common user role</description>
-            <privileges>
-                <privilege>VIEW_USER</privilege>
-            </privileges>
-        </role>
-        <role>
-            <name>CUSTOM_ROLE</name>
-            <description>Deployment role</description>
-            <privileges>
-                <privilege>Deploy</privilege>
-            </privileges>
-        </role>
-        <!-- Will lead to ConfigurationException: Embedded roles overriding is forbidden: [ADMIN] -->
-        <!--<role>
-            <name>adMIN</name>
-            <description>Deployment role</description>
-            <privileges>
-                <privilege>Deploy</privilege>
-            </privileges>
-        </role> -->
-    </roles>
-    <privileges>
-        <!-- Will lead to ConfigurationException: Embedded privileges overriding is forbidden: [ADMIN_CREATE_REPO] -->
-        <!-- <privilege>
-            <name>ADMIN_CREATE_REPO</name>
-            <description></description>
-        </privilege> -->
-        <privilege>
-            <name>EXAMPLE_PRIVILEGE</name>
-            <description>This is an example of how you could add some custom privilege. Feel free to use it by name at
-                the REST API side to access restriction purposes.
-            </description>
-        </privilege>
-    </privileges>
+
+    <!-- special treatment role -->
+    <role>
```

Comment:
I believe the requirement was to drop the wrapper class, but not the actual `<roles/>` XML wrapper. Could you please change this?

Reply:
+1